README.md

```diff
@@ -29,6 +29,40 @@ Tutorials in the **Intro-Full** directory cover
 
 All the tutorial folders can be built using either the `Makefile` or the CMake `CMakeLists.txt` file in each folder.
 
+## CMake
+
+CMake can build against an installed Kokkos library or download one automatically using `FetchContent`.
+
+Without any Kokkos already installed in an exercise directory, one can run the following:
+
+```shell
+cmake -B build_dir . # -DKokkos_* options
+cmake --build build_dir
+```
+
+Kokkos options are described in [CMake options](https://kokkos.org/kokkos-core-wiki/keywords.html).
+
+For example, OpenMP CPU exercises can be built as:
+```shell
+cmake -B build_openmp -S . -DKokkos_ARCH_NATIVE=ON -DKokkos_ENABLE_OPENMP=ON
+cmake --build build_openmp --parallel
```

Comment:
Not sure we should showcase the use of `--parallel`, so users don't just copy/paste and use on systems login nodes.